zvariant/src/type.rs

```diff
@@ -140,6 +143,26 @@ where
     }
 }
 
+/// A helper type to serialize a tuple whose elements implement [DynamicType] but not [Type].
+///
+/// This is required because tuples already have an implementation of [DynamicType] via the blanket
+/// implementation of [DynamicType] where `T: Type`, but that results in a bound of [Type] on each
+/// element, which is stronger than needed for serializing.
+#[derive(Debug, Copy, Clone)]
```

Comment:
Why `Copy`? We shouldn't assume `T` to be `Copy`.

Reply:
It's a `#[derive]` so it doesn't require that `T` is `Copy`.  Technically it might be better to derive all the traits (`Eq` and so forth) but the rest are less useful.

Reply:
TIL that derives don't add the impl if the generics don't satisfy the the underlying trait.